sycl/test-e2e/Plugin/sycl-ls-gpu-default-any.cpp

```diff
@@ -21,6 +21,6 @@
 // is present.
 // The test crashed on CUDA CI machines with the latest OpenCL GPU RT
 // (21.19.19792).
```

Comment:
Shall this comment be updated?

Reply:
Thanks for pointing this out. I've removed this out of date comment.